readme.md

```diff
@@ -13,26 +13,20 @@ plugin it depends on.
 
 Therefore after cloning this project please:
 
-1. Install JXcore from [here](http://jxcore.com/downloads) and install Node.JS from [here](http://nodejs.org) (for desktop testing)
-2. If you havent installed Cordova (jx install cordova -g) on Windows / (sudo jx install cordova -g) on Unix
-3. Create a cordova project : Follow the instructions [here](https://cordova.apache.org/docs/en/4.0.0/guide_cli_index.md.html) to install node.js, Cordova and whatever platforms you require (e.g. cordova plafrom add android and cordova platform add ios).
-
-4. Go into your cordova application folder(postcard) and 'git clone https://github.com/jxcore/jxcore-cordova'
-5. Run 'cordova plugins add jxcore-cordova'
-6. Then run 'cordova platform add android' (or what ever platform you are using) to add the platform support  
-
-7. From command line goto postcardapp/www/jxcore folder, and run 'jx install'
-
-8. Go to file explorer and goto postcardapp/www/jxcore/node_modules and search for *.gz" and delete all *.gz files
-
-9. Now you can run 'cordova run android' from postcardapp folder (or whatever platform you are using) and it should just run.
-
-
-
-
-
-
-
-
+1. Install JXcore from [here](http://jxcore.com/downloads) 
+2. Install Cordova 
+ 1. Windows - `jx install cordova -g`
+ 2. Linux - `sudo jx install cordova -g` 
+ 
+```
+curl https://codeload.github.com/thaliproject/postcardapp/zip/story_0 > thali.zip
+unzip thali.zip
+cd postcardapp-story_0
+cordova platform add android
+cd www/jxcore
+jx npm install --production
+find . -name "*.gz" -delete
```

Comment:
Perfect! :+1:   Would love to see people use cmder by default on Windows quite honestly!